Content.Shared/Sound/Components/EmitSoundOnActionComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Sound.Components;
+
+/// <summary>
+/// Simple sound emitter that emits sound on InstantAction
+/// </summary>
+[RegisterComponent]
+public sealed partial class EmitSoundOnActionComponent : BaseEmitSoundComponent
+{
+    /// <summary>
+    ///     Whether or not to mark an interaction as handled after playing the sound. Useful if this component is
+    ///     used to play sound for some other component with on-use functionality
+    /// </summary>
+    /// <remarks>
+    ///     If false, you should be confident that the interaction will also be handled by some other system, as
+    ///     otherwise this might enable sound spamming, as use-delays are only initiated if the interaction was
+    ///     handled.
+    /// </remarks>
+    [DataField("handle")]
```

Comment:
If not specified the yaml tag will default to the camelcase version of the field's name
```suggestion
    [DataField]
```